mysql-test/suite/innodb/r/instant_alter_charset.result

```diff
@@ -78,60 +78,6 @@ alter table all_texts
 convert to charset latin1 collate latin1_general_ci,
 algorithm=instant;
 drop table all_texts;
-create table all_binaries (
-a tinyblob,
-b blob,
-c mediumblob,
-d longblob,
-e varbinary(150),
-f binary(150)
-) engine=innodb;
-alter table all_binaries modify a tinytext, algorithm=instant;
-ERROR 0A000: ALGORITHM=INSTANT is not supported. Reason: Cannot change column type INPLACE. Try ALGORITHM=COPY
```

Comment:
Where were these tests moved to?
Yes, changing collation on non-indexed columns should trivially be allowed with `ALGORITHM=INSTANT`. But we should still test that.